_pages/docs/scenarios/idnex.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
is the file name `_pages/docs/scenarios/index.md`?

Reply:
lol yeah, good catch